paasta_tools/cli/cmds/validate.py

```diff
@@ -209,6 +224,60 @@ def path_to_soa_dir_service(service_path):
     return soa_dir, service
 
 
+def validate_tron(service_path):
+    soa_dir, service = path_to_soa_dir_service(service_path)
+    returncode = True
+
+    if soa_dir.endswith('/tron'):
+        # Makes it possible to validate files in tron/ rather than service directories
+        # TODO: Clean up after migration to services is complete
+        cluster = service
+        soa_dir = soa_dir[:-5]
```

Comment:
These 5 characters are the /tron right? 
How about just .strip('/tron')?

Reply:
strip removes any combination of the characters, so it'll remove other stuff if the service name ends in one of those.